### PR TITLE
Use enrollments with the same validator cycle as config.yaml

### PR DESCRIPTION
**Pull request by kchulj**
Issue [#1062](https://github.com/bpfkorea/agora/issues/1062): When running the integration test on a local node, the following message was showing:
```console
$ dub build
$ cd tests/system/node/2/
$ ../../../../build/agora
[...]
[main(ZbnT) INF] Listening for requests on http://0.0.0.0:2826/
Task terminated with uncaught exception: Request failure to http://node-6:6826 after 5 attempts
Task terminated with uncaught exception: Request failure to http://node-7:7826 after 5 attempts
Task terminated with uncaught exception: Request failure to http://node-4:4826 after 5 attempts
Task terminated with uncaught exception: Request failure to http://node-5:5826 after 5 attempts
Task terminated with uncaught exception: Request failure to http://node-3:3826 after 5 attempts
2020-07-27 09:15:36,758 Info [agora.consensus.ValidatorSet] - Invalid pre-image data: The pre-image has a invalid hash value. Pre-image: { enroll_key: 0xb20da9cfbda971f3f573f55eabcd677feaf12f7948e8994a97cdf9e570799b71631e87bb9ebce0d6a402275adfb6e365fdb72139c18559a10df0e5fe4bae08eb, hash: 0x148006017980424589fc1a7f360f815ef1bcce71a3bd0b581f7a126a57449cb058bba5eca3642cac3e93c15ebd2f2d54d206e6ee3c7fc06b8a59e597d5e02840, distance: 11 }
```
This is due to a mismatch of the pre-images after they have been propagated. The `validator_cycle` in the config files are set to 20, and the enrollments in the [Test Genesis Block](https://github.com/bpfkorea/agora/blob/v0.x.x/source/agora/consensus/data/genesis/Test.d) were using 1008. Now, they both use a cycle of 20.

The enrollments have been re-generated and the hard-coded genesis block has been replaced to temporarily fix this error.

### CODE REVIEW

**Review by AndrejMitrovic — APPROVED** [review]

**Comment by AndrejMitrovic** [comment]
Ah one thing, this PR introduced whitespace changes.

Please change your editor to strip trailing lines, and amend the commit.

File: source/agora/consensus/data/genesis/Test.d [hunk 1]
```diff
@@ -51,64 +51,70 @@ public immutable Block GenesisBlock = {
...
+            // Node 4
             Enrollment(
-                Hash(`0x46883e83778481d640a95fcffd6e1a1b6defeaac5a8001cd3f99e17576b809c` ~
-                     `7e9bc7a44c3917806765a5ff997366e217ff54cd4da09c0c51dc339c47052a3ac`),
-                Hash(`0x34404dca05b15527046f83f03da31a9684d03d2d0204d3cb5849d13fb3c989d` ~
-                     `df4f2f04f388355e12f4ed2976e0966d5e02a6f7f1d735ca52e192abbb019c79c`),
-                1008,
-                Signature(`0x0c2fd79ec6deb83eb2baa4da7bdfe37595bb42a768dcd66abc62943023fa806` ~
-                          `8dc8b89135fe3f5df9e2815b9bdb763c41b8b2dab5911e313acc82470c2147422`)
+                Hash(`0x46883e83778481d640a95fcffd6e1a1b6defeaac5a8001cd3f99e17576b809c` ~
```
Comment (AndrejMitrovic) [inline source/agora/consensus/data/genesis/Test.d]: Trailing white space here.

**Review by Geod24 — APPROVED** [review]
Apart from the whitespace issue, LGTM

**Review by Geod24 — APPROVED** [review]

**Review by TrustHenry — APPROVED** [review]
LGTM

**Comment by AndrejMitrovic** [comment]
Well done @kchulj!

**Comment by Geod24** [comment]
Wait does this work ? https://app.circleci.com/pipelines/github/bpfkorea/agora/3883/workflows/320afea0-8fe7-452a-aef3-c1b7ca4d1300/jobs/4071
I'm seeing a lot of this error in the CI.

**Comment by AndrejMitrovic** [comment]
I think that's a different error? Probably related to what I mentioned in https://github.com/bpfkorea/agora/issues/1062#issuecomment-664804315

**Pull request merged**